FAQ.md

```diff
@@ -72,6 +72,19 @@ Firecracker supports Linux host and guest operating systems with kernel versions
 4.14 and above. The long-term support plan is still under discussion. A leading
 option is to support Firecracker for the last two Linux stable branch releases.
 
+### What toolchains are supported by Firecracker?
```

Comment:
To me, the gist of this section feels more at home in the `Getting the Firecracker Binary` from `docs/getting-started.md` (and maybe we should rename that section to `Building the Firecracker Binaries` btw). `README.md` mentions how to build using the container, and sends ppl to `getting-started.md` for more build information, so everything should be easy to find.